Keas.Mvc/Services/IdentityService.cs

```diff
@@ -199,6 +225,10 @@ public async Task<string> BulkLoadPeople(string ppsCode, string teamslug)
                     {
                         var personResult = await GetOrCreatePersonFromKerberos(kerbResults.ResponseData.Results[0].UserId, team.Id);
                         newpeople += personResult.peopleCount;
+                        if (personResult.Person == null)
+                        {
+                            warning.Append($"IAM ID {id.IamId} failed to save.");
+                        }
                     }
```

Comment:
@jcubbage Add an else with a warning here, but you can probably only have the IAM id as the kerb may not have been found. This should be the else to the `if (kerbResults.ResponseData.Results.Length > 0 && contactResult.ResponseData.Results.Length > 0)`

Reply:
@jcubbage That will not catch IAM users that PPS returns, but that we don't find.
You would need it on the else of this if (If we want to report these)
`                    var kerbResults = await clientws.Kerberos.Search(KerberosSearchField.iamId, id.IamId);
                    var contactResult = await clientws.Contacts.Get(id.IamId);

                    if (kerbResults.ResponseData.Results.Length > 0 && contactResult.ResponseData.Results.Length > 0)
                    {
                        var personResult = await GetOrCreatePersonFromKerberos(kerbResults.ResponseData.Results[0].UserId, team.Id);
                        newpeople += personResult.peopleCount;
                        if (personResult.Person == null)
                        {
                            if (kerbResults.ResponseData.Results.Length > 0)
                            {
                                warning.Append($" IAM ID: {id.IamId}, Kerb Id: {kerbResults.ResponseData.Results.First().UserId} failed to save.");
                            }
                            else
                            {
                                warning.Append($" IAM ID {id.IamId} failed to save.");
                            }
                        }
                    }`